pkg/microservice/aslan/core/service/handler/router.go

```diff
@@ -53,6 +53,7 @@ func (*Router) Inject(router *gin.RouterGroup) {
 		k8s.POST("", GetServiceTemplateProductName, gin2.UpdateOperationLogStatus, CreateServiceTemplate)
 		k8s.PUT("", gin2.UpdateOperationLogStatus, UpdateServiceTemplate)
 		k8s.PUT("/yaml/validator", YamlValidator)
+		k8s.PUT("/:serviceName/yaml/view", YamlViewServiceTemplate)
```

Comment:
Does the API require permission control? 
Please take a look @mouuii

Reply:
Add it after the permission policy is optimized @mouuii